### PR TITLE
Fix for cstor_pool_scale test

### PR DESCRIPTION
**Pull request by shashank855**
**What this PR does / why we need it**:
- `kubectl patch` command for scale up/down the pool deployment worked for openshift but failing in pipeline. Instead of patch we can use `kubectl scale`.

### CODE REVIEW

File: apps/percona/chaos/cstor_pool_scale/test.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@
           register: cvr_name1  
 
         - name: Scaling-down cstor-pool
-          shell: kubectl patch deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --type=json -p='[{"op":"add", "path":"/spec/replicas", "value":"0"}]'
+          shell: kubectl scale deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --replicas=0 
           register: result1
           failed_when: "'patched' not in result1.stdout"
```
Comment (dargasudarshan) [inline apps/percona/chaos/cstor_pool_scale/test.yml]: @shashank855 The check condition should be changed here as you are scaling 'patched' string wont be there in stdout right ?

File: apps/percona/chaos/cstor_pool_scale/test.yml [hunk 2]
```diff
@@ -117,7 +117,7 @@
           register: cvr_name2   
 
         - name: Scaling-up cstor-pool
-          shell: kubectl patch deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --type=json -p='[{"op":"add", "path":"/spec/replicas", "value":"1"}]'
+          shell: kubectl scale deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --replicas=1
           register: result2
           failed_when: "'patched' not in result2.stdout"
```
Comment (dargasudarshan) [inline apps/percona/chaos/cstor_pool_scale/test.yml]: Same as above comment

File: apps/percona/chaos/cstor_pool_scale/test.yml (hunk 1 above)
Comment (shashank855) [reply apps/percona/chaos/cstor_pool_scale/test.yml]: @dargasudarshan I made the change.

File: apps/percona/chaos/cstor_pool_scale/test.yml (hunk 2 above)
Comment (shashank855) [reply apps/percona/chaos/cstor_pool_scale/test.yml]: @dargasudarshan Made the change.

**Pull request merged**